webserver.py

```diff
@@ -91,30 +91,37 @@ def save(self, slug):
                                [slug, json_game_state, json_game_state])
 
     def initialize(self):
-        for seed_id in GAME_CONFIG['seeds']:
-            if self.data['seedCounts'].get(seed_id) is None:
-                self.data['seedCounts'][seed_id] = 0
         for resource in GAME_CONFIG['starting_resources']:
             if self.data['resources'].get(resource) is None:
-                self.data['resources'][resource] = GAME_CONFIG['starting_resources'][resource]
+                if GAME_CONFIG['starting_resources'][resource] > 0:
```

Comment:
I don't think you need this "if starting resources > 0" check.  (though it doesn't hurt)

The reason is, if I set the game config to say `starting_resources: {'apple_seed': 0}` my intention may be that the user sees apple seeds initially, but just has zero of them.  with this `if` check in place, including a 0 starting resource is no different from leaving it out entirely